src/main/java/org/opensearch/knn/index/codec/backward_codecs/KNN9120Codec/KNN9120DerivedSourceReaders.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.knn.index.codec.backward_codecs.KNN9120Codec;
```

Comment:
Moving this to backwards_codecs because we need to minimize number of readers derived source relies on. In the new version, we only need DocValues and the vectors.